umake/frameworks/ide.py

```diff
@@ -181,7 +181,7 @@ def post_install(self):
         comment = self.description + " (UDTC)"
         categories = "Development;IDE;"
         create_launcher(self.desktop_filename,
-                        get_application_desktop_file(name=self.name,
+                        get_application_desktop_file(name=self.description,
```

Comment:
It's to have the exact same name as the tools themselves generate. For example, Idea Ultimate has this wizard that runs when it's started up for the first time, and it can overwrite our .desktop file, so this way the name won't change.